src/theme/css/variables.css

```diff
@@ -60,4 +60,4 @@
         
         --chapter-hover-bg: var(--theme-hover);
     }
-}
+}
```

Comment:
Do end files with a blank line
```suggestion
}

```

Reply:
Ok, next time